mautic/mautic
# Add text for the rebuilding contacts time for segment. The text "Tota…

…l time:" was left for the sum of the rebuilding time for all segments



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [No ]
| New feature/enhancement? (use the a.x branch)      | [ Yes]
| Deprecations?                          | [ No]
| BC breaks? (use the c.x branch)        | [ No]
| Automated tests included?              | [ No] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 





#### Description:

Add text for the rebuilding contacts time for segment. The text "Total time:" was left for the sum of the rebuilding time for all segments.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Run: $ php bin/console mautic:segments:update




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11419"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
